### PR TITLE
Update 4.1.4

### PR DESCRIPTION
**Pull request by Jrice1317**
- Updated version and hash
- Updated pinnings
- Updated doc url
- Updated home url
- Added abs.yaml

Jira: https://anaconda.atlassian.net/browse/PKG-588
https://github.com/dpgaspar/Flask-AppBuilder/tree/v4.1.4

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -70,13 +70,14 @@ test:
     - pip check
 
 about:
-  home: http://flaskappbuilder.pythonanywhere.com/
+  home: https://flaskappbuilder.pythonanywhere.com/
   license_file: LICENSE
   license: BSD-3-Clause
   license_family: BSD
   summary: Simple and rapid application development framework, built on top of Flask. includes detailed security, auto CRUD generation for your models, google charts and much more.
+  description: Simple and rapid application development framework, built on top of Flask.
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: The summary is longer than the description. It should be the opposite

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (Jrice1317) [reply recipe/meta.yaml]: Okay, thanks!

**Review by varlackc — APPROVED** [review]
LGTM

**Review by ryanskeith — APPROVED** [review]
LGTM

**Pull request merged**